app/src/main/java/com/android/voyageur/ui/overview/Overview.kt

```diff
@@ -87,7 +105,6 @@ fun OverviewScreen(
   val context = LocalContext.current
   val isConnected = status === ConnectionState.Available
 
-  Log.e("RECOMPOSE", "OverviewScreen recomposed")
   LaunchedEffect(isLoadingUser, isLoadingTrip) { isLoading = isLoadingUser || isLoadingTrip }
   LaunchedEffect(trips) {
     if (trips.isNotEmpty()) {
```

Comment:
While the functionality is correct, the code uses multiple LaunchedEffect blocks to handle state changes, making it harder to track loading logic. Consider consolidating these side effects into a single LaunchedEffect that reacts to all relevant states (e.g., isLoadingUser, isLoadingTrip, and trips). For example, you could rely on a single ViewModel-aggregated isLoading state and respond to trips availability in one place. This would simplify the composable’s logic and enhance maintainability by providing a central point for handling state changes instead of spreading logic across multiple LaunchedEffect calls.